frontend/repositories/tag/apiTagRepository.ts

```diff
@@ -0,0 +1,34 @@
+import ApiService from '@/services/api.service'
+import { TagRepository } from '~/domain/models/tag/tagRepository'
+import { TagItem } from '~/domain/models/tag/tag'
+
+export interface TagItemResponse {
+  id: number,
+  text: string,
+  project: string
```

Comment:
should be a number?

Reply:
project = projectId 
projectId is defined as string in comment methods f.e. - I wanted to keep that scheme.

Reply:
no problem. i was just wondering